tests/unit_tests/test_ustar_cp/test_computeStandardisedScores.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+import numpy as np
+
+@pytest.mark.parametrize(
+    "input_data, expected_xNormX",
+    [
+        #
+        # Test 1: 2x3 matrix, no NaNs
+        #
+        #   x = [[1,2,3],
+        #        [4,5,6]]
+        #
+        #   For each column (with only 2 values):
+        #     median => (value1 + value2)/2
+        #     iqr    => 75th - 25th percentile => typically 1.5 for each col
+        #
+        #   => xNorm => row0 => [-1, -1, -1]
+        #               row1 => [ 1,  1,  1]
+        #   => xNormX => [1, 1]
+        #
```

Comment:
```suggestion
        #
        # Test 1: 2x3 matrix, no NaNs
```
Comments removed as inconsistent with test definition